src/core/settings.js

```diff
@@ -15,6 +15,7 @@ export const create = () => {
         name: "settings-closed",
         callback: () => {
             setAccessibleLayer(true);
+            gmi.setStatsScreen("settings");
```

Comment:
Think we are missing a test here.

Reply:
Will add! I'm summing up all the tests I need to write